Exiled.API/Enums/DamageType.cs

```diff
@@ -68,6 +68,10 @@ public enum DamageType
         Scp,
 
         /// <summary>
+        /// Damage dealt by SCP-096.
```

Comment:
This is a breaking change, do not add new values above existing ones in an enum.

Reply:
Oh okay i need to add at the ends it's that ?